sessions/js-structure/quiz-app/global.css

```diff
@@ -0,0 +1,59 @@
+:root {
+	--primary: #ffffff;
+	--secondary: #ff4a11;
+	--secondary-light: #fddcd2;
+	--water: #f3f5f9;
+	--black: #252629;
+	--ff-primary: sans-serif;
+	--ff-serif: serif;
+}
+
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+h1,
+h2,
+h3 {
+	line-height: 1.2;
+}
+
+body,
+h1,
+h2,
+h3,
+p {
+	margin: 0;
+}
+
+body {
+	line-height: 1.5;
+	font-family: var(--ff-primary);
+	color: var(--black);
+}
+
+button {
+	font-size: var(--fs-400);
```

Comment:
I'm not sure if this variable still exists?

Reply:
```suggestion
```

🫠